src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -81,103 +210,12 @@ def generate_FAD_1_default_config(self, filename):
         config = {
             'registrationRequests': devices,
             'conditionalRegistrationData': conditionals,
-            'grants': grants,
+            'grantRequests': grants,
             'ppas': ppas,
             'pals': pals,
```

Comment:
`palRecords` is slightly more readable.

GRA.17 in #429 is also using `ppaRecord` (singular because there's only one in that test case); I can see arguments for each name but we should try to keep them consistent across test cases. Let me know if you prefer to change GRA.17 or this code.

Reply:
I changed the name here, eventhough, for me, the word "record" was more for the dump data.